apps/web/src/components/layout/components/LocalStudioSidebar/LocalStudioSidebar.tsx

```diff
@@ -1,33 +1,26 @@
 import { WithLoadingSkeleton } from '@novu/novui';
-import { css, cva } from '@novu/novui/css';
-import { styled } from '@novu/novui/jsx';
+import { IconCloudQueue } from '@novu/novui/icons';
 import { useDiscover } from '../../../../studio/hooks/useBridgeAPI';
+import { useStudioNavigate } from '../../../../studio/hooks/useStudioNavigate';
+import { Aside } from '../../../nav/Aside';
 import { LocalStudioSidebarContent } from './LocalStudioSidebarContent';
-
-const Aside = styled(
-  'aside',
-  cva({
-    base: {
-      position: 'sticky',
-      top: '0',
-      zIndex: 'auto',
-      backgroundColor: 'transparent',
-      borderRight: 'none',
-      width: '[272px]',
-      height: 'full',
-      p: '50',
-      bg: 'surface.panel',
-      overflowY: 'auto',
-    },
-  })
-);
+import { SidebarFooterButton } from './SidebarFooterButton';
 
 export const LocalStudioSidebar: WithLoadingSkeleton = () => {
   const { isLoading, data } = useDiscover();
 
+  const navigate = useStudioNavigate();
+
+  const goToCloudDashboard = () => {
+    navigate(window.location.origin, {});
+  };
```

Comment:
@antonjoel82 I think here we can just do `window.open(FRONT_END_URL (https://dashboard.novu.co)` in a new tab?

I want to make sure that the user understands it's a different application

Reply:
Sure, but what is `FRONT_END_URL`? I don't see anything similar in `web`